data/Chinwenma.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "Chinwenma Okorie",
+  "bio": " **FrontEnd Developer** | Open Source enthusiast | Blogger | Tech enthusiast",
+  "tags":[ " Open Source", "NextJs", "ReactJs",  "JavaScript", "Tailwind", "Html5", "CSS3"],
+  "socials": [
+      {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/Ethel_Omma"
+    },
+      {
+       "icon": "FaGit",
+      "url": "https://github.com/Chinwenma"
+    },
+          {
+      "icon": "FaLinkedin",
+       "url": "https://github.com/Chinwenma"
+
+    },
```

Also, remove this trailing comma! 

```suggestion
    }
```